src/app/components/pages/guide-section-article-page/guide-section-article-page.component.ts

```diff
@@ -57,7 +57,7 @@ import {UserResolver} from '../../../resolvers/user/user.resolver';
 
       <div class="ui mobile only grid">
         <guide-card
-          [fadedTitle]="(guideSectionArticle | async)?.translatedText?.title"
+          [fadedTitle]="(guideSectionArticle | async)?.section.translatedText?.title"
```

Comment:
Here the chaining comes *again*, would be way nicer to have one function to call instead, but as I recall you've mentioned that the performance for that is *horrible*, is that still true for current versions of Angular?